### ISSUE TITLE
Usage of described_class

### ISSUE THREAD

**Issue by fenec** [lelylan/betterspecs#194]
https://relishapp.com/rspec/rspec-core/docs/metadata/described-class

I recently started to use `described_class` instead of explicitly stating the class name:
```ruby
let(:obj) { described_class.new }
vs
let(:obj) { Klass.new }
```
I didn't find any guidelines about this, so I wanted to discuss the pros and the cons of each approach. 
An advantage of the former is that it looks cleaner. And I believe that the name of the method is self-evident even for those who are not familiar with RSpec's syntax. 
What do you think, guys?

**Comment by nicknish** [comment]
Great thought! I prefer to `described_class` as it helps reduce redundancy in your code. 

Similar to `subject`, it helps DRY up your test suite. Imagine using `Klass.new` all over your spec suite. If you were to ever change the name of your class, you'd have to change it everywhere.

**Comment by akz92** [comment]
I agree! It also makes the code more reusable as you don't need to worry with the new class's name.
